yarn.lock

```diff
@@ -7018,11 +7018,6 @@ lodash@^3.10.0:
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-3.10.1.tgz#5bf45e8e49ba4189e17d482789dfd15bd140b7b6"
   integrity sha1-W/Rejkm6QYnhfUgnid/RW9FAt7Y=
 
-lodash@^4.14.2:
```

Comment:
I'm curious why this happened.
I will verify this locally

Reply:
lodash is a direct dependency to ReactNativeCameraKit I think
Check in `node_modules/react-native-camera-kit/node_modules/lodash`

Reply:
yes this part looks good